.github/workflows/actions_build.yml

```diff
@@ -87,18 +92,47 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK ${{ matrix.java }}
+      - id: setup-jdk-17
```

Comment:
Just to make sure, these changes don't have any meaning other than unifying some logic to be similar to `armeria`'s. Am I understanding correctly?   (since `lint` was already running on java 17 before this PR)

(I think also we can remove `lint.strategy.matrix` also)

Reply:
Thanks! removed it. 😉